lib/AL_USDMaya/AL/usdmaya/nodes/wrapProxyShape.cpp

```diff
@@ -0,0 +1,362 @@
+//
+// Copyright 2017 Animal Logic
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.//
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "AL/usdmaya/nodes/ProxyShape.h"
+
+#include <boost/python/args.hpp>
+#include <boost/python/def.hpp>
+#include <boost/python.hpp>
+
+#include "AL/usdmaya/Utils.h"
+
+#include "maya/MBoundingBox.h"
+#include "maya/MFnDagNode.h"
+#include "maya/MFnDependencyNode.h"
+#include "maya/MDagModifier.h"
+#include "maya/MDGModifier.h"
+#include "maya/MSelectionList.h"
+
+#include "pxr/base/tf/pyEnum.h"
+#include "pxr/base/tf/pyResultConversions.h"
+
+#include <memory>
+
+//using namespace std;
+//using namespace boost::python;
+//using namespace boost;
+
+
+using AL::usdmaya::nodes::ProxyShape;
+using boost::python::reference_existing_object;
+using boost::python::object;
+
+namespace {
+  struct MBoundingBoxConverter
+  {
+    // to-python conversion of const PxrUsdMayaXformOpClassification.
+    // Decided NOT to register this using boost::python::to_python_converter,
+    // in case pxr registers one at some point
+    static PyObject* convert(const MBoundingBox& bbox)
```

Comment:
eventually we should move these out to a new home for more general utilities - and don't currently have a place for python related bits and pieces. We're currently reorganising the code, so happy to hold off until then

Reply:
makes sense... let me know when / if you find a better home for this!  Ideally, we would want to register this from the USD side of things, with boost::python::to_python_converter, so that MBoundingBox objects are automagically converted to python objects...